components/Printing.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+export default function Printing() {
+  const handleSubmission = () => {};
```

Comment:
Create new issue to work on sending data to backend. Collab with @spenpal2000 to receive data appropriately.